model/riscv_insts_vext_total.sail

```diff
@@ -0,0 +1,9655 @@
+/* ****************************************************************** */
+/* This file specifies the instructions in the vector set.      */
+
+
+/* ****************************************************************** */
```

Comment:
Missing suitable copyright header matching the rest of the repo as per #146

Reply:
It seems that there is no package called headache in CentOS, Could you please give me some advice?

Reply:
No need for headache, just copy the template from other files, and write a list of the contributors to each file added ??